xptifw/unit_test/xpti_correctness_tests.cpp

```diff
@@ -167,43 +166,62 @@ TEST(xptiCorrectnessTest, xptiNotifySubscribersForDefaultTracePointTypes) {
   xpti::payload_t p("foo", "foo.cpp", 1, 0, (void *)13);
   xptiForceSetTraceEnabled(true);
 
-  uint8_t StreamID = xptiRegisterStream("test_foo");
+  uint8_t StreamID = xptiRegisterStream("test_foo2");
   auto Result = xptiRegisterCallback(
       StreamID, (uint16_t)xpti::trace_point_type_t::graph_create, tpCallback);
+  ASSERT_EQ(static_cast<xpti::result_t>(Result),
```

Comment:
It's strange that you needed a type-cast here but no in the other instances below.